Odata-docs/client/v6/using_blazor_wasm_with_odata_client.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Blazor Wasm with OData Client"
+description: ""
+
+author: elokerio
+ms.author: elokerio
+ms.date: 1/18/2021
+ms.topic: article
+ 
+---
+# How To Use Blazor Web Assembly with OData Client 
+**Applies To**: [!INCLUDE[appliesto-odataclient](../../includes/appliesto-odataclient-v6.md)]
+
+# Getting Started
+Start by creating a Blazor WebAssembly project.
+
+Open Visual Studio and click on Create a new project then select the Blazor App template and click Next. Configure your project by providing the project name and the location path then click Create. On the Create Project Page that appears, specify the target framework that you want to use and select Blazor WebAssembly App then click Create. A blazor web assembly project will be created like below. 
+
+![SAMPLE BLAZOR WEB ASSEMBLY PROJECT](../../assets/2021-02-02-blazor-wasm-app-sample.png)
+
+There are two ways in which you can use OData Client with blazor web assembly: 
+
+* You can use the Microsoft OData Connected Service
+* You can POCO's
```

Comment:
Maybe you could expand the abbreviation here, or say "You can use plain-old C# objects (POCOs)"

Reply:
fixed

Reply:
resolved